client/app/components/QueryEditor.jsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { map } from 'lodash';
+import { Tooltip } from 'antd';
```

Comment:
While webpack supposed to get rid of unused code, we did import specific components (`antd/lib/tooltip`) when we used it in other places. @kravets-levko do you remember why?

Reply:
fixed